docs/book/common_code/images/activity_machinedeployment_reconciliation.drawio

```diff
@@ -0,0 +1 @@
+<mxfile modified="2019-03-20T16:25:35.373Z" host="www.draw.io" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36" etag="lHfJqVL0CJpnDvWAJo1N" version="10.5.1" type="google"><diagram id="L4uQc6gKZ2xOVQujygMW" name="Page-1">7VzJlps4FP0aLysHEJOXSQ1J+pycTncturPEIAxdGNFCjk1/fUsgzCC7rCozmTiLxHrMT/fdeyUUFuB+s/+MnST4hjwYLTTF2y/Aw0LTVHVp039YJCsjABSRNQ49HqsCz+F/kAcVHt2GHkwbOxKEIhImzaCL4hi6pBFzMEa75m4+ippXTZw1FALPrhOJ0b9CjwRF1DaUKv4FhuugvLKq8C0bp9yZB9LA8dCuFgKPC3CPESLFr83+HkYse2VeiuOeTmw93BiGMZE5AP725fnr0/brU/zwb4wJihL/jzt+lp9OtOUPzG+WZGUG6FlosmnjU0A2EY2p9Cd9mIRtT4mDyTNxCNvuh1F0jyKE8wOBkv9hOxOMXmBti+/zLfzqEBO4P/lY6iFZFGYQbSDBGd2lPEAxikM4wkze3FW9BXgPBLWOKmMOx8f6cOIqhfQHz+IbMqpJZNSjEONNhEmA1ih2oscqWs80S05I8fgxCtcxja0QIWhDN8DY+8gAzk6SwLiI8AKyX806zSzO/qZB5YNRNn+wZtl42DdaGW8VD8Lu/vW+og+LttiF53FH0bOG5JX9wPG+xzBySPizeR/HupIf+h2F9A4PmLGVBmRUswWG4rb4QRUeaLqdrLZbwnZIT19G1VrX0exmhb5tf/qjuIMKnIeUvB+vQMDrZ0hy/nKDMIYeTCKUbWB+n2j1D+PYNp4x2sYe9Dhgd0FI4HPi5P2/o7LQxLODXQ5SnWHKRzE5xhktLvHpH7d3LjFb+de1DyKbHHqpTid6X3Si3+hEgk6AJJ0YvdBJu261CjdnGKWrIjYElETOKrdiKYxoydI+yx2JGzAv9bTQzIim6dOKhs01OfRFvaYDtFlt0/P1PEqhWq2Eg6OFqhwpVKuvQjWFLshgelmtOrxGI+iTI6VLUHLtdWtI1q31KhpKN5E1ufjiqm6eFkhXtXCqu6ZFvdNaIES+n8JeiMESULnO1d2NtimBOB8R4Urty7v4FdQdGOOru31TdwmWsCRZYjmIuuvG0Oq+FFBSlS/chym7WSrkzoYVY/H3qWa8Sou2gnYxxH9Cv34stwJMN+dlEHRgjW4QyhmtWjfGqFeD0Gn1N2q/ooL+q38pWf2qJmUSLmSDgxaUyLKGZgNVnLObq9PMW98hDmnOIB4RWuP4zybSDMO+NvepipOh+ZWOis8J4VKKI1YRcl9+Z4c90BEtCVE8X6MqcMwEnKoqzhPerOppxL+bUkYijbdObQOldZ03Tm0bQ0xtq+LotwPjzEyb4jESgqyjmTaxXWbmmI1ytDyeXxYHxze/LENBssNlTZdyNR0bZmNpDG2YxfHzXA3zKDBSpGDUrzk22zQ0fXOsifMBjufVtcWDFDUezIVY8UOKRoodpl2hn+sQe4wEwxTGv86srWmP74U1cQR+GZtMX4jO0kXZAUeo5vxb2RMM0jFHWLYcR1y8zEMBRgtjPZhbTVwZUH+Jw9/r1JdvtA1q4YBn8DZYsBije1dNfCF/Td51NNNRgvrdlNGtd9VVwbtKMUhnNX5bVNAniMDrtNKPc9WWbUMDpDA1IecKRP/Dpecb1QMuPW1NOaNEsxSiE2Z1UCkC2hVKUYdsAC51of1KiqXKTdB2VrviPP5NUroDEZCbU+tXUmxFdoptSqIijmcwdDyWPIKOyQmbJPHmqBq2oo2tGdc9fBlt6h3IOk99FK2xq1VQQ6nNbQDTK4zMCajNUrs6rSkzUYNlmOadtU3noSlC6Y+uKbo4aLxpigQZDCMVhj22VOjiOPUmFZLomKpU6Ob1S4U4XqaAcAhcZ6JIvMRoF89BPgQ6GF8+xOHh9OWjy0KX/i+ug+iFZYyuF+IY9aYX3cFIt0fQCxNcv16II164h+42X9CTZrErKsFcVuoArd17R17DD7pQRxcXAF9GENMfYpxnAPMke4y1UAe0v8eitOu1q1XrwoWGWKmjiwuZK0pofF+lcpZz5QjDmhxHiKuDJ8cR7651e3K13pb43mpdvNAQtV7a4nd/UI1BYszPqVn6qf7p/oNqtFl9/q7ogeorguDxfw==</diagram></mxfile>
```

Comment:
This doesn't show what is expected to happen if the label selector doesn't match

Reply:
I added a requeue to the diagram.